CHANGELOG.md

```diff
@@ -43,7 +43,8 @@ title: Changelog
 * `<cc-pricing-table>`: add text content to `<cc-button>` elements (toggle State and add buttons) so that they can be identified by assistive technologies.
 * `<cc-zone-input>`: add `tabindex="0"` to the list container so that people can focus and scroll the list without having the change their selection.
 * `resizeObserver`: add `window.requestAnimationFrame` in the resize observer callback to mitigate the `resize observer loop limit exceeded issue`.
-* stories: fix miscellaneous stories not being passed to `enhancedStoryNames`.
+* stories: fix miscalleneous stories not being passed to `enhancedStoryNames`. 
+* `<cc-tile-status-codes>`: fix JavaScript errors in case initial loading fails (error state).
```

Comment:
you can check the error in prod, just open the `error` story of the component:
![image](https://user-images.githubusercontent.com/100240294/189328700-8131771c-9395-44f0-ad16-6f7a217d457b.png)

Reply:
:clap: Nice catch!